src/java/org/apache/cassandra/cql3/UntypedResultSet.java

```diff
@@ -228,7 +228,7 @@ protected Row computeNext()
                         try (ReadExecutionController executionController = pager.executionController();
                              PartitionIterator iter = pager.fetchPageInternal(pageSize, executionController))
                         {
-                            currentPage = select.process(iter, nowInSec, true).rows.iterator();
+                            currentPage = select.process(iter, nowInSec, true, ClientState.forInternalCalls()).rows.iterator();
```

Comment:
`FromPager` is only used by `QueryProcessor#executeInternalWithPaging()`, so this should be safe.